frontend/src/modules/EditZing/Types/CourseInfo.ts

```diff
@@ -39,3 +40,15 @@ export interface CourseStudentDataResponse {
     groups: ResponseGroup[]
   }
 }
+
+export type TemplateDataResponse = {
+  success: boolean
+  data: {
+    id: string
+    name: string
+    type: 'group' | 'student'
+    subject: string
+    body: string
+    modifyTime: Date
+  }[]
+}
```

Comment:
Is this the same as `EmailTemplatesResponse` in `@core/Types`? I think this whole file may be deprecated soon in favor of centralizing shared types across different components.